crates/ruff_linter/src/rules/flake8_bugbear/rules/function_call_in_argument_default.rs

```diff
@@ -62,9 +71,9 @@ impl Violation for FunctionCallInDefaultArgument {
     fn message(&self) -> String {
         let FunctionCallInDefaultArgument { name } = self;
         if let Some(name) = name {
-            format!("Do not perform function call `{name}` in argument defaults")
+            format!("Do not perform function call `{name}` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable")
         } else {
-            format!("Do not perform function call in argument defaults")
+            format!("Do not perform function call in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable")
```

Comment:
I'm not a huge fan of bugbear's error messages which tend to be very long (https://github.com/PyCQA/flake8-bugbear/blob/907e0dd29a99818591a604d4557c70ea33204712/bugbear.py#L1660-L1667). I prefer to put that level of detail in our documentation, or make it evident via the proposed fix. But I do think this one can be a bit clearer. Wdyt @zanieb?